wannierberri/__path.py

```diff
@@ -88,7 +97,37 @@ def __init__(self,system,k_list=None,k_nodes=None,length=None,dk=None,nk=None,la
                     self.breaks.append(self.K_list.shape[0]-1)
             self.K_list = np.vstack( (self.K_list, k_nodes[-1]) )
             self.labels[self.K_list.shape[0]-1] = labels[-1]
+        else:
+            self.K_list=np.array(k_list)
+            assert  self.K_list.shape[1]==3, "k_list should contain 3-vectors"
+            assert  self.K_list.shape[0]>0, "k_list should not be empty"
+            for var in 'k_nodes','length','nk','dk':
+                if locals()[var] is not None:
+                    warning("k_list was entered manually, ignoring {}".format(var))
+            self.labels={} if labels is None else labels
+            self.breaks=[] if breaks is None else breaks
+        self.div = np.shape(self.K_list)[0]
         self.breaks=np.array(self.breaks,dtype=int)
+  
+    def sphere(self,type_sphere,r1,r2,ntheta,nphi,origin):
```

Comment:
What about removing `type_sphere` argument and using `self.sphere(r1,r1,ntheta,nphi,origin)` for the sphere case?

Reply:
Done